fiona/ogrext.pyx

```diff
@@ -37,40 +38,7 @@ from libc.string cimport strcmp
 from cpython cimport PyBytes_FromStringAndSize, PyBytes_AsString
 
 
-log = logging.getLogger("Fiona")
-
-# Mapping of OGR integer field types to Fiona field type names.
-#
-# Lists are currently unsupported in this version, but might be done as
-# arrays in a future version.
-
-FIELD_TYPES = [
-    'int',          # OFTInteger, Simple 32bit integer
-    None,           # OFTIntegerList, List of 32bit integers
-    'float',        # OFTReal, Double Precision floating point
-    None,           # OFTRealList, List of doubles
-    'str',          # OFTString, String of ASCII chars
-    None,           # OFTStringList, Array of strings
-    None,           # OFTWideString, deprecated
-    None,           # OFTWideStringList, deprecated
-    'bytes',        # OFTBinary, Raw Binary data
-    'date',         # OFTDate, Date
-    'time',         # OFTTime, Time
-    'datetime',     # OFTDateTime, Date and Time
-    'int',          # OFTInteger64, Single 64bit integer
-    None,           # OFTInteger64List, List of 64bit integers
-    ]
-
-# Mapping of Fiona field type names to Python types.
-FIELD_TYPES_MAP = {
-    'int':      int,
-    'float':    float,
-    'str':      text_type,
-    'date':     FionaDateType,
-    'time':     FionaTimeType,
-    'datetime': FionaDateTimeType,
-    'bytes':    bytes,
-   }
+log = logging.getLogger(__name__)
```

Comment:
This is way more correct and makes testing the logging more simple (just configure the root logger).